src/data/changelogHooks.ts

```diff
@@ -1,10 +1,10 @@
 import { CHANGELOG_ROOT } from '@/utils/apiUrlUtil';
-import { useQuery } from 'react-query';
+import { useQuery } from '@tanstack/react-query';
 import { Changelog } from '@/api/types/changelogTypes';
 import { get } from '@/api/axios';
 
 export const changelogsQueryKeys = {
-  changelogs: 'changelogs',
+  changelogs: ['changelogs'],
```

Comment:
https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#query-keys-and-mutation-keys-need-to-be-an-array